src/components/Forms/ClientInfo.js

```diff
@@ -44,18 +46,33 @@ export default function FormPropsTextFields() {
           flexDirection: "row",
           justifyContent: "flex-start",
           alignItems: "center",
-          gap: 2, // Add spacing between the elements
+          // gap: 2, // Add spacing between the elements
```

Comment:
If we don't need this, can we remove this comment?

Reply:
Ok